world-server/src/network/Server.ts

```diff
@@ -12,6 +12,10 @@ export class Server implements Messages.MsgHandler {
     constructor() {
         this.connCounter = 0;
         this.sockets = [];
+
+        // Only to bypass the build errors
+        this.httpServer = http.createServer();
+        this.wsServer = new websocket.server();
```

Comment:
There is probably a way better solution for this then instantiating dummy servers

Reply:
The alternative is that we call `init()` in the constructor, which doesn't see like a bad idea for our use case. I'll implement it and  let me know if this shouldn't be done.

Reply:
UPDATE: This solution doesn't work, reverting to the dummy server solution for now.